src/Twilio/Http/BearerToken/SystemNetBearerTokenHttpClient.cs

```diff
@@ -0,0 +1,142 @@
+        /// <returns>Twilio response</returns>
+        public override Response MakeRequest(BearerTokenRequest request)
+        {
+            try
+            {
+                var task = MakeRequestAsync(request);
+                task.Wait();
+                return task.Result;
+            }
+            catch (AggregateException ae)
+            {
+                // Combine nested AggregateExceptions
+                ae = ae.Flatten();
+                throw ae.InnerExceptions[0];
+            }
+        }
+
+        /// <summary>
+        /// Make an asynchronous request
+        /// </summary>
+        /// <param name="request">Twilio response</param>
+        /// <returns>Task that resolves to the response</returns>
+        public override async Task<Response> MakeRequestAsync(BearerTokenRequest request)
+        {
+            var httpRequest = BuildHttpRequest(request);
+            if (!Equals(request.Method, HttpMethod.Get))
+            {
+                if (request.ContentType == null)
+                    request.ContentType = EnumConstants.ContentTypeEnum.FORM_URLENCODED;
+
+                if (Equals(request.ContentType, EnumConstants.ContentTypeEnum.JSON))
+                    httpRequest.Content = new StringContent(request.Body, Encoding.UTF8, "application/json");
+
+                else if (Equals(request.ContentType, EnumConstants.ContentTypeEnum.SCIM))
+                    httpRequest.Content = new StringContent(request.Body, Encoding.UTF8, "application/scim");
+                
+                else if(Equals(request.ContentType, EnumConstants.ContentTypeEnum.FORM_URLENCODED))
+                    httpRequest.Content = new FormUrlEncodedContent(request.PostParams);
+            }
+
+            this.LastRequest = request;
+            this.LastResponse = null;
+
+            var httpResponse = await _httpClient.SendAsync(httpRequest).ConfigureAwait(false);
+            var reader = new StreamReader(await httpResponse.Content.ReadAsStreamAsync().ConfigureAwait(false));
+
+            // Create and return a new Response. Keep a reference to the last
+            // response for debugging, but don't return it as it may be shared
+            // among threads.
+            var response = new Response(httpResponse.StatusCode, await reader.ReadToEndAsync().ConfigureAwait(false), httpResponse.Headers);
+            this.LastResponse = response;
+            return response;
+        }
+
+        private HttpRequestMessage BuildHttpRequest(BearerTokenRequest request)
+        {
+            var httpRequest = new HttpRequestMessage(
+                new System.Net.Http.HttpMethod(request.Method.ToString()),
+                request.ConstructUrl()
+            );
+
+            var authBytes = request.AccessToken;
+            httpRequest.Headers.Authorization = new AuthenticationHeaderValue("Bearer", authBytes);
+
+            httpRequest.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
```

Comment:
`application/json` this should come from API instead of setting it here `application/json` for all APIs.

Reply:
for the time being we support only application/json should we add support for all types? please suggest